src/otx/algorithms/common/adapters/mmcv/configurer.py

```diff
@@ -260,6 +262,8 @@ def configure_fp16(cfg: Config):
                     cfg.fp16 = fp16_config
                     opts = dict()
                 cfg.optimizer_config.update(opts)
+            elif is_hpu_available():
```

Comment:
This change can break mmaction integration, since it requires modified train() on our side.
I'd propose to configure hooks here for all devices, and remove this section from train()

Reply:
Done. Could you check if it is correct? https://github.com/openvinotoolkit/training_extensions/pull/2574/commits/751a4e2c0e917a1d5db86d9f6c193c9d496b3e22

Reply:
It looks fine now